src/incubator/WheelPicker/Item.tsx

```diff
@@ -30,56 +31,28 @@ export default ({
   // value,
   itemHeight,
   onSelect,
-  offset
+  activeIndex
 }: InternalProps) => {
-  const itemOffset = index * itemHeight;
-  const opacity = Animated.interpolate(offset, {
-    inputRange: [itemOffset - itemHeight, itemOffset, itemOffset + itemHeight],
-    outputRange: [0.5, 1, 0.5]
-  });
-
-  const rotateXValue = useMemo(() => {
-    return Animated.interpolate(offset, {
-      inputRange: [
-        itemOffset - 2.5 * itemHeight,
-        itemOffset,
-        itemOffset + 2.5 * itemHeight
-      ],
-      outputRange: [-85, 0, -85]
-    });
-  }, [itemHeight]);
-
-  const rotateX = concat(rotateXValue, 'deg');
-
-  const scale = useMemo(() => {
-    return Animated.interpolate(offset, {
-      inputRange: [
-        itemOffset - 2 * itemHeight,
-        itemOffset,
-        itemOffset + 2 * itemHeight
-      ],
-      // outputRange: [0.8, 1, 0.8], // with scale change
-      outputRange: [1, 1, 1],
-    });
-  }, [itemHeight]);
+  
+  const isActive = activeIndex === index;
 
   const selectItem = useCallback(() => onSelect(index), [index]);
 
+  const color = isActive ? Colors.primary : Colors.grey20;
```

Comment:
we will need to find a way to transition the color using animation. it's look bulky at the moment.